app/repositories/meteors.repository.js

```diff
@@ -0,0 +1,10 @@
+import axios from "axios";
+import { getApiUrl } from "../utils/getApiUrl.js";
+
+export const getMeteors = async (startDate, endDate) => {
+  const url = getApiUrl(startDate, endDate);
+
+  const response = await axios.get(url);
```

Comment:
use params for forming query parameters 

https://stackoverflow.com/a/53501339/12225394

Reply:
Am I mistaken, or adding query params is the very next task [#9](https://godelonline.sharepoint.com/Functions/LearningAndDevelopment/SitePages/Node---Proxy-Server.aspx?csf=1&web=1&e=Z6bZS6&OR=Teams-HL&CT=1732611317249&clickparams=eyJBcHBOYW1lIjoiVGVhbXMtRGVza3RvcCIsIkFwcFZlcnNpb24iOiI0OS8yNDEwMjAwMTMxOCIsIkhhc0ZlZGVyYXRlZFVzZXIiOmZhbHNlfQ%3D%3D&CID=5a831324-1a88-4207-96aa-41cca15bce8e#task-9-add-parameters-to-the-query)?

Reply:
but you already added them
![Uploading image.png…]()
Even if it’s in the next task, it’s already in the code, so I can leave a comment :)

Reply:
if you don’t mind I'll make these changes in the next PR